### PR TITLE
Publish OpenTelemetry Schema File 1.5.0

### PR DESCRIPTION
**Pull request by tigrannajaryan**
Please merge this after https://github.com/open-telemetry/opentelemetry-specification/pull/1801 is merged.

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
Please merge this after https://github.com/open-telemetry/opentelemetry-specification/pull/1801 is merged.

**Comment by tigrannajaryan** [comment]
@open-telemetry/docs-approvers please review and merge.

**Review by flands — APPROVED** [review]

**Pull request merged**
